test/src/sw/redis++/zset_cmds_test.hpp

```diff
@@ -134,7 +134,7 @@ void ZSetCmdTest<RedisInstance>::_test_range() {
 
     _redis.zadd(key, s.begin(), s.end());
 
-    REDIS_ASSERT(_redis.zcount(key, UnboundedInterval<double>{}) == s.size(),
+    REDIS_ASSERT(_redis.zcount(key, UnboundedInterval<double>{}) == static_cast<long long int>(s.size()),
```

Comment:
`zcount` returns signed integer, a container length is an unsigned integer. Assuming the length of the container is relatively small in this synthetic test, we can use this simple cast.